halo2_backend/src/plonk/keygen.rs

```diff
@@ -41,7 +41,7 @@ where
 /// Generate a `VerifyingKey` from an instance of `CompiledCircuit`.
 pub fn keygen_vk_v2<'params, C, P>(
```

Comment:
How about renaming `keygen_vk_v2` to `keygen_vk_from_compiled_circuit` or `keygen_vk`?
I think it is good to remove `_v2` suffix.
What do you think? @ed255

Reply:
Thanks a lot for spotting this!  When doing this PR I only searched for `V2` (with uppercase V) so I missed the `_v2` suffixes :P
I've renamed `keygen_vk_v2` to `keygen_vk` and `keygen_pk_v2` to `keygen_pk`.

Although these function names are also defined in `halo2_proofs` a user should never import both, because it's not useful to mix the legacy `halo2_proofs` with the frontend-backend split API.